src/components/Navigation.tsx

```diff
@@ -0,0 +1,40 @@
+import classNames from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+export const Navigation: React.FC = () => {
+  return (
+    <nav
+      // eslint-disable-next-line max-len
+      className="navbar is-light is-fixed-top is-mobile has-shadow has-navbar-fixed-top"
+      data-cy="Nav"
+    >
```

Comment:
you should fix linter errors, not disable them 😄 
```suggestion
    <nav
      className="
        navbar 
        is-light 
        is-fixed-top 
        is-mobile 
        has-shadow 
        has-navbar-fixed-top
      "
      data-cy="Nav"
    >
```

Reply:
Hii, I'll fix it 😄